system_metrics_collector/src/system_metrics_collector/periodic_measurement_node.cpp

```diff
@@ -92,6 +94,44 @@ bool PeriodicMeasurementNode::SetupStart()
   return true;
 }
 
+rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn
+PeriodicMeasurementNode::on_activate(const rclcpp_lifecycle::State &)
+{
+  RCLCPP_DEBUG(this->get_logger(), "on_activate");
+  auto const ret = Start();
+  return ret ? rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn::SUCCESS :
```

Comment:
Consider adding `using SUCCESS = rclcpp_lifecycle::node_interfaces::LifecycleNodeInterface::CallbackReturn::SUCCES` at the top of the file to make this more readable. I.e. there should not be two ways to do a similar thing.

Reply:
Will do.